apps/dask-gateway/templates/traefik/service.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: v1
 kind: Service
 metadata:
   name: {{ include "dask-gateway.traefikName" . }}
```

Comment:
this service name should be mounted as an environment variable in the singleuser pods, since the ip address can change and users won't have a way to look it up